text-to-sql-Shuchi/app.py

```
@@ -0,0 +1,121 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)